src/ray/gcs/gcs_server/gcs_resource_manager.cc

```diff
@@ -167,8 +212,7 @@ void GcsResourceManager::UpdateFromResourceReport(const rpc::ResourcesData &data
     UpdateNodeNormalTaskResources(node_id, data);
   } else {
     if (node_resource_usages_.count(node_id) == 0 || data.resources_available_changed()) {
-      const auto &resource_changed = MapFromProtobuf(data.resources_available());
-      SetAvailableResources(node_id, ResourceSet(resource_changed));
+      SetAvailableResources(node_id, MapFromProtobuf(data.resources_available()));
     }
```

Comment:
If normal task resource is temporarily disabled, then it should call SetAvailableResources() here even with gcs actor scheduler enabled.

Reply:
test_schedule_actor_and_normal_task
test_schedule_many_actors_and_normal_tasks
test_actor_distribution_balance
test_worker_lease_reply_with_resources

These four tests are relying on normal task resources update. They have to be skipped for now.